deployment/constructor_params/lynx/root.yml

```diff
@@ -17,6 +18,8 @@ constants:
   FORTY_THOUSAND_TOKENS_IN_WEI_UNITS: 40000000000000000000000
   TEN_MILLION_TOKENS_IN_WEI_UNITS: 10000000000000000000000000
   YEAR_2025: 1735689600
+  PENALTY_DEFAULT: 1000 # 10%
+  PENALTY_INCREMENT: 500 # 5% increment
```

Comment:
My guess that for testnet we want those values to be bigger.

Reply:
> My guess that for testnet we want those values to be bigger.

How much would you choose for `default` and `increment`? @vzotova 

We were not sure about the numbers, so we decided (if I remember correctly) to have low quantities so we could play with a wider range of penalizations.

Reply:
considering testnet I'd do values like 50% default and 25% for increment, to make it more noticeable for anyone who uses testnet. for mainnet - yeah, smaller is better

Reply:
I agree, but let's not redeploy because of that. In any case, we will measure reward amounts with scripts. We can change it later if needed.